src/events.ts

```diff
@@ -0,0 +1,37 @@
+import Emittery from "emittery"
+import { CID } from "./common/cid.js"
+import { DistinctivePath, Partition, Partitioned } from "./path/index.js"
+
+
+export type Events = {
+  "fileSystem:local-change": { root: CID; path: DistinctivePath<Partitioned<Partition>> }
+  "fileSystem:published": { root: CID }
+}
```

Comment:
We are prefixing "fileSystem" here in case we add other types of event emitters in the future? If so, what would you think about namespacing at the top-level on the program. Something like:

```typescript
program.events.fileSystem.on("local-change", ({ path, root }) => {
  ...
})
```

Reply:
Done 👍

Reply:
Actually now that I think more about adding other events, I'm changing this to:
```ts
program.fileSystem.on("...")
```
Because we already have `program.fileSystem`